src/classes/USER_UserService_TEST.cls

```diff
@@ -0,0 +1,247 @@
+        System.assertEquals(USER_UserService.ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS, errorSettingsSysAdmin.Error_Notifications_To__c);
+    }
+
+
+    /*********************************************************************************************************
+    * @description Tests if User is the owner of Scheduled Jobs and calls the methods that logs an error and sends an email.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testHandleScheduledJobsSubmittedByInActiveIsFrozenSysAdmin() {
+        List<User> newUsers = createUsers();
+        Set<Id> sysAdminIds = new Map<Id, User>(newUsers).keySet();
+
+        List<CronTrigger> jobs = [SELECT Id FROM CronTrigger];
+        for(CronTrigger eachJob : jobs) {
+            System.abortJob(eachJob.Id);
+        }
+
+        System.runAs(newUsers[0]) {
+            UTIL_MasterSchedulableHelper.setScheduledJobs();
+        }
+
+        List<CronTrigger> scheduledJobs = [SELECT Id, OwnerId, CronExpression, CronJobDetail.Name, CreatedBy.Name, CreatedBy.IsActive 
+                                            FROM CronTrigger
+                                            WHERE CronJobDetail.JobType = :USER_UserService.JOB_TYPE_SCHEDULED_APEX];
+
+        // Freeze a User
+        List<UserLogin> currentUserLogin = [SELECT IsFrozen, UserId FROM UserLogin WHERE UserId = :newUsers[0].Id];
+        currentUserLogin[0].IsFrozen = true;
+        Database.update(currentUserLogin, false);
+
+
+        Test.startTest();
+        USER_UserService.handleScheduledJobsSubmittedByInActiveIsFrozenSysAdmin(sysAdminIds);
+        Test.stopTest();
+
+        if(!scheduledJobs.isEmpty()) {
+            System.assertEquals(9, scheduledJobs.size());
+        }
+
+        List<Error__c> retrievedErrorTypeAfterEmailSent = [SELECT Email_Sent__c FROM Error__c WHERE Object_Type__c = :USER_UserService.OBJECT_TYPE_USER];
+        System.assertNotEquals(null, retrievedErrorTypeAfterEmailSent);
+        System.assertEquals(true, retrievedErrorTypeAfterEmailSent[0].Email_Sent__c);
+    }
+
+
+    /*********************************************************************************************************
+    * @description Tests logging and Error__c and sending and email to System Administrators.
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testNotifySysAdminsAboutInActiveIsFrozenSysAdminUser() {
+        Test.startTest();
+        USER_UserService.notifySysAdminsAboutInActiveIsFrozenSysAdminUser();
+        Test.stopTest();
+
+        List<Error__c> retrievedErrorTypeAfterEmailSent = [SELECT Email_Sent__c FROM Error__c WHERE Object_Type__c = :USER_UserService.OBJECT_TYPE_USER];
+        System.assertNotEquals(null, retrievedErrorTypeAfterEmailSent);
```

Comment:
this assert just says the soql returned a list, which it always will.  it doesn't assert that the list isn't empty.

Reply:
This assert was to minimize repetition of the checks. However, this method needs to implement the checks in the testLogInActiveIsFrozenUserError and testSendInActiveFrozenUserEmail test methods.